docs/api/input.md

```diff
@@ -99,6 +101,27 @@ import CSSProps from '@site/static/usage/v7/input/theming/css-properties/index.m
 
 <CSSProps />
 
+## Migrating from Legacy Input Syntax
+
+A simpler input syntax was introduced in Ionic 7.0. This new syntax reduces the boilerplate required to setup an input, resolves accessibility issues, and improves the developer experience.
+
+While developers can continue using the legacy syntax, we recommend migrating as soon as possible.
+
+
+### Using the Modern Syntax
+
+Using the modern syntax involves two steps:
+
+1. Removing `ion-label` and using the `label` property on `ion-input` instead. The placement of the label can be configured using the `labelPlacement` property on `ion-input`.
+2. Moving input-specific properties from `ion-item` on to `ion-input`. This includes the `counter`, `counterFormatter`, `fill`, and `shape` properties, as well as the `helper` and `error` slots.
```

Comment:
Should we call out that the `helper` and `error` slots should be replaced with the `helperText` and `errorText` props here too? I'm good either way since that could be seen as too redundant.

Reply:
I updated the migration in https://github.com/ionic-team/ionic-docs/pull/2682/commits/99da9dfa83ed691ccdec7b556c994e320a757a81 to clarify this